### PR TITLE
Release prep v3.2.0

### PR DESCRIPTION
**Pull request by swarna04**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/AEPTarget.md [hunk 1]
```diff
@@ -325,14 +346,14 @@ static func retrieveLocationContent(_ requestArray: [TargetRequest], with target
 
 This API sets the custom visitor ID for Target.
 
-    This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall or when the resetExperience API is used.
+    This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall, upon privacy status update to opted out or when the resetExperience API is used.
```
Comment (sbenedicadb) [inline Documentation/AEPTarget.md:349]: caleb would say you're missing the oxford comma here 😜

File: Documentation/AEPTarget.md [hunk 2]
```diff
@@ -399,11 +420,53 @@ ACPTarget.getThirdPartyId({id, err in
...
+[AEPMobileTarget setTntId:@"f741a5d5-09c0-4931-bf53-b9e568c5f782.35_0"];
+```
+
 ## getTntId
 
-This API gets the Target user identifier. Target returns the tntId with a successful call to `prefetchContent`, which is then persisted in the SDK.
+This API gets the Target user identifier. 
 
-    This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall or when the resetExperience API is used.
+The tntId is returned in the network response from Target after a successful call to `prefetchContent` API or `retrieveLocationContent` API, which is then persisted in the SDK. The persisted tntId is used in subsequent Target requests until a different tntId is returned from Target or a new tntId is set using `sentTntId` API.
```
Comment (sbenedicadb) [inline Documentation/AEPTarget.md:469]: typo: set using `sentTntId` API => set using `setTntId` API

File: Documentation/AEPTarget.md [hunk 3]
```diff
@@ -413,13 +476,95 @@ This API gets the Target user identifier. Target returns the tntId with a succes
...
+```swift
+static func getSessionId(_ completion: @escaping (String?, Error?) -> Void)
+```
+
+  - *completion* : invoked with the Target `sessionId` value or `nil` if there was an error retrieving it.
 
 #### Example
 
 ```swift
     Target.getTntId { id, err in
```
Comment (sbenedicadb) [inline Documentation/AEPTarget.md:566]: change to `getSessionId`

**Review by sbenedicadb — CHANGES_REQUESTED** [review]
couple minor things that need to be fixed

**Comment by swarna04** [comment]
@sbenedicadb I've updated the pr, can you please take a look. Ty!

**Review by sbenedicadb — APPROVED** [review]
looks good

**Pull request merged**